packages/superset-ui-connection/src/SupersetClientClass.ts

```diff
@@ -40,6 +44,7 @@ export default class SupersetClientClass {
     this.credentials = credentials;
     this.csrfToken = csrfToken;
     this.csrfPromise = undefined;
+    this.fetchRetryOptions = fetchRetryOptions;
```

Comment:
Maybe `this.fetchRetryOptions = { ...DEFAULT_FETCH_RETRY_OPTIONS, ...fetchRetryOptions }` and     `fetchRetryOptions = {}` in `L33` in case people only want to override one of the options.

Reply:
good call